cli/cliutils/cliutils.go

```diff
@@ -214,6 +215,49 @@ func GetDockerAuth(domain string) (auth dockerclient.AuthConfiguration, err erro
 		}
 	}
 
+	// try to load cred from configured credential store
+	var cfgPath string
+	if dockerCfgPath := os.Getenv("DOCKER_CONFIG"); dockerCfgPath != "" {
```

Comment:
Why didnt you use the docker auth from cred store function in go-dockerclient? "https://github.com/fsouza/go-dockerclient/blob/main/auth.go#L296"

Reply:
In the current version of go-dockerclient we're using this functionality isn't available. Can I update version to the latest - 1.7.2?

Reply:
Yes, I think that would be a more appropriate way to address the problem.